db/migrate/20160422114747_copy_community_id_from_people_to_emails.rb

```diff
@@ -0,0 +1,23 @@
+class CopyCommunityIdFromPeopleToEmails < ActiveRecord::Migration
+  def up
+    exec_update(
+      ["UPDATE emails e",
+       "LEFT JOIN people p ON p.id = e.person_id",
+       "SET e.community_id = -1",
+       "WHERE p.community_id = -1"].join(" "),
+      "Set community_id -1",
+      []
+    )
+  end
```

Comment:
Why not just set the `NULL` `community_id`s in emails table to -1. Could copying from `people` table possibly leave some emails having `community_id` as `NULL`?